IMS.BL/Product.cs

```diff
@@ -4,14 +4,9 @@ namespace IMS.BL
 {
     public class Product
     {
-        public Product()
-        {
-            ProductId = Guid.NewGuid();
-        }
-        
-        public Guid ProductId { get; private set; }
-        public string ProductName { get; set; }
-        public decimal ProductPrice { get; set; }
-        public int ProductQuantity { get; set; }
+        public int Id { get; set; }
```

Comment:
why did you change to int from Guid ?? you still can have UUID in the sqlserver and its alternatives on any Database